nav2_behavior_tree/nav2_tree_nodes.xml

```diff
@@ -387,7 +387,11 @@
       <input_port name="error_codes_to_check">Error codes to check, user defined</input_port>
     </Condition>
 
-
+    <Condition ID="IsFollowingPath">
```

Comment:
Docs: need a configuration guide page for this new BT node + add to the Navigation Plugins page